eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -390,7 +393,7 @@ stages:
                 ArtifactPath: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
       - job: PublishDocsToNightlyBranch
         dependsOn: PublishPackages
-        condition: or(eq(variables['SetDevVersion'], 'true'), and(eq(variables['Build.Reason'],'Schedule'), eq(variables['System.TeamProject'], 'internal')))
+        condition: and(eq('${{ parameters.SkipServicePublishDocsMs }}', 'true'), or(eq(variables['SetDevVersion'], 'true'), and(eq(variables['Build.Reason'],'Schedule'), eq(variables['System.TeamProject'], 'internal'))))
```

Comment:
Given this is a template parameter we should do a template if instead of doing a runtime condition check.